src/BloomExe/Book/BookCompressor.cs

```diff
@@ -264,13 +264,20 @@ private static string GetMetaJsonModfiedForTemplate(string path)
 			return meta.Json;
 		}
 
-		private static void InsertReaderStylesheet(XmlDocument dom)
+		private static void InsertReaderStylesheets(XmlDocument dom)
 		{
+			var head = XmlUtils.GetOrCreateElement(dom, "html", "head");
 			var link = dom.CreateElement("link");
-			XmlUtils.GetOrCreateElement(dom, "html", "head").AppendChild(link);
 			link.SetAttribute("rel", "stylesheet");
 			link.SetAttribute("href", "readerStyles.css");
 			link.SetAttribute("type", "text/css");
+			head.AppendChild(link);
+
+			link = dom.CreateElement("link");
+			link.SetAttribute("rel", "stylesheet");
+			link.SetAttribute("href", "file:///android_asset/book support files/android.css");
```

Comment:
It seems like we're being misleadingly specific by calling out "android". ".bloomd" isn't meant to be ".blooma".
Similarly, it seems awkward for Bloom Desktop to know the path inside of BloomReader in which to get this file.